packages/live-server/package.json

```diff
@@ -12,14 +12,14 @@
   ],
   "author": "Tapio Vierros",
   "dependencies": {
-    "chokidar": "1.6.0",
+    "chokidar": "3.1.0",
```

Comment:
why upgrade this here but not in core? I am a big fan of using tools like https://www.npmjs.com/package/npm-check-updates to perform actions like this across the monorepo

Reply:
Hey @bmuenzenmeyer I walked my way through the directories and it seems like I missed the core with chokidar. Thanks for the note and providing a link to check the monorepro a better way.

Reply:
no problem! turns out it was good to omit for now anyways, see #1069 😊